### PR TITLE
cluster-import-flow

### PR DESCRIPTION
**Pull request by kamleshverma1**

### CODE REVIEW

**Comment by kamleshverma1** [comment]
@gnehapk Please review this PR

File: src/api/Brick.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
        "value": "brick1"
     }, {
         "name": "brick2",
-		"value": "brick2"
+		    "value": "brick2"
     }, {
         "name": "brick3",
         "value": "brick3"
```
Comment (gnehapk) [inline src/api/Brick.json]: Remove unnecessary space.

File: src/commons/components/generate-form/generate-form.js [hunk 2]
```diff
@@ -12,44 +12,42 @@
...
                     $scope.manipulateData = function() {
-                        var keys = Object.keys($scope.formAttributes),
-                            len = keys.length,
-                            requestData = {},
-                            i;
+                        var requestData = {};
 
-                        for (i = 0; i < len; i++) {
-                            requestData[keys[i]] = $scope.formAttributes[keys[i]].value;
+                        for (var i = 0; i < $scope.formAttributes.length ; i++) {
```
Comment (gnehapk) [inline src/commons/components/generate-form/generate-form.js]: Why are re-declaring variable i again.
move $scope.formAttributes.length  to len variable.

File: src/commons/services/utils.js [hunk 3]
```diff
@@ -43,12 +43,12 @@
                 return response.data;
             });
         };
-        
-        vm.getListOptions= function() {
+
+        vm.getListOptions= function(listType) {
             var getListRequest;
             var getListRequest = {
```
Comment (gnehapk) [inline src/commons/services/utils.js]: remove re-declaration of getListRequest.

File: src/commons/components/generate-form/generate-form.js (hunk 2 above)
Comment (gnehapk) [inline src/commons/components/generate-form/generate-form.js]: please refer http://stackoverflow.com/questions/8452317/do-loops-check-the-array-length-every-time-when-comparing-i-against-array-length

**Pull request merged**